### ISSUE TITLE
Unable to use with yarn workspace

### ISSUE THREAD

**Issue by nekosaur** [applitools/eyes-storybook#22]
This package can not currently be used with yarn workspaces, since it depends on finding files inside the project's `node_modules` directory. But with yarn workspaces all packages are hoisted to the root of the workspace. The folder structure is as follows:

```
root
  node_modules
  packages
    storybook
```

This means that using `process.cwd()` here https://github.com/applitools/eyes-storybook/blob/5df92590b8ba4fa57718289fa4ad24e36f45293f/src/cli.js#L32

will make it look in the wrong folder. I would guess the simplest solution would be to make `packagePath` configurable through the `applitools.config.js` file?

When I manually edited the code in `cli.js` to use `process.cwd() + '/../../'` everything worked fine, so this is the only thing stopping it from working.

**Comment by danielschwartz85** [comment]
@nekosaur sory for the late response, We do support yarn as far as I see.
What error do you get ? also from what lib are you running ```npx eyes-storybook``` ?
Thansk,

**Comment by nekosaur** [comment]
I get the following error

```bash
$ yarn test:snapshot
yarn run v1.17.3
$ eyes-storybook
Using @applitools/eyes-storybook version 2.7.25.

ENOENT: no such file or directory, open 'D:\foo\packages\storybook\node_modules\@storybook\core\package.json'
error Command failed with exit code 1.
```

where `test:snapshot` script executes `eyes-storybook` as you can see. But `package.json` exists in the root workspace node_module folder, not the node_modules folder of the storybook project, where I am executing the script (see folder structure in my first post).

**Comment by danielschwartz85** [comment]
Do you have ```@storybook\core``` package though in node_modules ?

**Comment by nekosaur** [comment]
It exists, but not in the path that `eyes-storybook` explicitly looks for it. Like I explained, all packages are being hoisted to the root `node_modules` folder because I am using yarn workspaces https://yarnpkg.com/lang/en/docs/workspaces/, so the `@storybook\core` package is not available in `foo\packages\storybook\node_modules`, but in `foo\node_modules`.

**Comment by danielschwartz85** [comment]
Sorry I still don't see how this can happen / can't reproduce this issue..

We need the @storybook package at the top level of your project i.e. we do not expect it to be in eyes-storybook's node_modules - this root dir should be the dir from which you run eyes.

Do you run eyes from ```foo/``` ?

Regardless note that you can try starting storybook server by your self and then starting eyes-storybook with -u parameter for setting the storybook url:
https://github.com/applitools/eyes-storybook/blob/master/README.md#advanced-configuration

**Comment by nekosaur** [comment]
I'm not sure I'm just terrible at explaining, but here's a minimal reproduction.

https://github.com/nekosaur/yarn-workspaces-eyes-storybook

Just cd to `packages/storybook` folder and run `yarn eyes-storybook`.

Running the storybook server separately and then using `-u` parameter does work, but it's not very convenient for CI purposes.

Also the url you posted seems to be pretty unrelated?

**Comment by danielschwartz85** [comment]
I am able to start eyes-storybook from ```yarn-workspaces-eyes-storybook``` dir but fail to start storybook server because Storybook's config dir is missing.

So basically I think the issue is that eyes-stroybook can/should be ran from the root dir but Storybook server cannot be started from it:
This fails:
```
cd yarn-workspaces-eyes-storybook/
npx start-stroybook
```
We need to add ```-c``` for telling Stroybook the config dir.

So long story short - If you add -c and run eyes from the root dir eyes should run ok:
```
cd yarn-workspaces-eyes-storybook/
npx eyes-storybook -c packages/storybook/.storybook
```

**Comment by nekosaur** [comment]
Using your `-c` workaround on the test repo, it seems to hang on starting the server. It doesn't get past this part.

```bash
⠙ Starting storybook server
```

Regardless of that workaround working or not, I would still classify this as a bug. I can't think of a reason why you try to manually read the `@storybook/core` package.json instead of requiring it.

Replacing this

https://github.com/applitools/eyes-storybook/blob/06e5d06fdbd075e6c4a7b3a5ea77123d1ae77727/src/startStorybookServer.js#L22-L26

with this

```js
const storybookPackage = require('@storybook/core/package.json');
const storybookVersion = storybookPackage.version;
```

solves the issue.

**Comment by danielschwartz85** [comment]
This looks good:
```const storybookPackage = require('@storybook/core/package.json');```
published in ```2.7.31```
Thanks.

**Comment by nekosaur** [comment]
Thanks! Can confirm it works in my project setup now 👍
